iree/task/task.h

```diff
@@ -244,7 +244,7 @@ typedef struct iree_task_call_closure_t {
   // contain embedded values. No lifetime management is performed by the task
   // system and it is required that users ensure that the memory referenced is
   // live until after the task has completed.
-  uintptr_t user_context;
+  void* user_context;
```

Comment:
nit: could reword the comment above this slightly

it current is
```
  // User-defined argument passed to task functions during invocation.
  // Opaque pointer-sized values that could point to user data structures or
  // contain embedded values.
```

("pointer-sized values" seems redundant when the type is now `void*`)

maybe reword to `Opaque pointer to a user-provided data structure`?